### ISSUE TITLE
Possible clarifications on OIDC instructions

### ISSUE THREAD

**Issue by mgibson323** [sensu/sensu-docs#3208]
### Description

We had a ticket with a customer who experienced an issue with OKTA authentication - and it turns out, the port they had in the redirect URL was incorrect (`8000` instead of `8080` - `8000` is the `web port` for their configuration). Our thinking was that perhaps it might be valuable to call out that the proper port to use here is the `API port` rather than the `web port`?

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [ ] update (add missing or refresh existing content)
- [x] improvement (improve docs content or format)

### Affected pages

https://docs.sensu.io/sensu-go/latest/operations/control-access/oidc-auth/#create-an-okta-application

### Expected behavior

We're wondering if there is value in making it more explicit that the `<api_url>` should also use the `<api_port>` in step 5 of these instructions? 

### Additional context

Tl;DR customer accidentally glossed over that it should be the API port, which led to a ticket and eventually a call with the customer to resolve. It's my opinion that this is already pretty clear so if this is not changed then it's completely understandable! 

* Browser(s) and version(s): 
* OS:
